src/main/java/org/terasology/ligthandshadow/componentsystem/controllers/ScoreSystem.java

```diff
@@ -110,11 +111,25 @@ private void checkAndResetGameOnScore(ActivateEvent event, EntityRef entity) {
             EntityRef heldFlag = getHeldFlag(player);
             if (checkIfTeamScores(baseTeamComponent, heldFlag)) {
                 incrementScore(baseTeamComponent);
-                if (redScore < LASUtils.GOAL_SCORE && blackScore < LASUtils.GOAL_SCORE) {
-                    resetRound(baseTeamComponent, heldFlag);
-                } else {
-                    resetLevel(player, baseTeamComponent, heldFlag);
+                resetRound(baseTeamComponent, heldFlag);
+                if (redScore == LASUtils.GOAL_SCORE) {
+                    logger.info("sending red won");
+                    sendEventToClients(new GameoverEvent(LASUtils.RED_TEAM));
                 }
+                if (blackScore == LASUtils.GOAL_SCORE) {
+                    logger.info("sending black won");
+                    sendEventToClients(new GameoverEvent(LASUtils.BLACK_TEAM));
+                }
+//                if (redScore < LASUtils.GOAL_SCORE && blackScore < LASUtils.GOAL_SCORE) {
```

Comment:
Generally you don't want to leave in commented out code like this. We've got revision history if needed :-)